insights/parsers/hammer_ping.py

```diff
@@ -1,21 +1,13 @@
 """
-hammer_ping - command ``/usr/bin/hammer ping``
-==============================================
+HammerPing - command ``/usr/bin/hammer ping``
+=============================================
 
 The hammer ping parser reads the output of ``hammer ping`` and turns it into
 a dictionary of service statuses in the ``data`` property.  Each service has
```

Comment:
@xiangce the `data` attribute is mentioned here, but not shown in the examples or defined as an attribute.  Would you mind to add docstrings for the two attributes `data` and `error`?